fbw-a32nx/src/wasm/systems/a320_systems/src/hydraulic/sfcc/slats_channel.rs

```diff
@@ -9,8 +9,9 @@ use systems::simulation::{
 };
 
 use uom::si::{angle::degree, f64::*};
+use uom::ConstZero;
 
-use super::sfcc::SlatFlapControlComputerMisc;
+use super::SlatFlapControlComputerMisc;
 
 pub struct SlatsChannel {
```

Comment:
```suggestion
pub(super) struct SlatsChannel {
```
Same with functions.